reference/kabusapi/WEBSOCKET_API.md

```
@@ -97,6 +165,6 @@ while (socket open):
 ## 参考
 
 * REST API ドキュメント: [REST_API.md](REST_API.md)
-* OpenAPI 仕様: `kabu_STATION_API.yaml`
+* 列挙値や詳細パラメータは REST API ドキュメント末尾の「列挙値リファレンス」を参照してください。
 
-以上。
+以上。
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)